open-craft/opencraft
# BB-2327: Fix color picker and Hero page

This PR fixes a few bugs in the color picker component and hero customization pages.

**Setup instructions:**
1. Checkout this branch, install API client.
2. Run Ocim & frontend.

**Testing instructions:**
42c376bc3e37a88494370be2bfd110fba8124e48, f26960da13d0c7d662fddf90d6662e5c1fabcd3e:
1. Check the pages that have the color picker and make sure they are being correctly displayed on HD, FullHD and 4k. It looks a little broken in mobile, but it's usable.
2. Check that the color preview has the correct color set after picking a color.
3. Try opening multiple color pickers at the same time, try to reproduce the issue from the previous implementation.
4. Check that the correct values are being saved into the backend.

From d8ceebf2eb92c8e92624810c1e03a1468406c05a onwards:
1. Go to the hero customization page.
2. Check that the styling is improved and that the fields are aligned with the container.
3. Check that it's possible to set empty values for the Hero text fields.
4. Check that the reset button works as expected :)

**Reviewer:**
